Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM rust:latest as builder
+
+WORKDIR ./shotover-proxy
+
+COPY ./ ./
+
+RUN cargo build
+
+FROM debian:bullseye-slim
+
+RUN apt-get update -y \
+    && apt-get install glibc-doc -y \
```

Comment:
why do we install glibc-doc?

Reply:
Not on `debian:bulleye-slim` by default.

Reply:
what is it though?

Reply:
Was a mistake leaving it in. Removed it as `debian:bullseye` includes by default.